test/select.jl

```diff
@@ -2939,4 +2939,82 @@ end
     end
 end
 
+@testset "Tables.AbstractRow interface" begin
```

Comment:
Maybe this should also be tested in other tests where we cover `DataFrameRow`?

Reply:
I have added more such tests (where I managed to track them down).